src/modules/popUpDom.js

```diff
@@ -0,0 +1,113 @@
+import getData from './getdata.js';
+import { getCommentData, addCommentInput } from './postComment.js';
+
+const popUpContainer = document.querySelector('.popUp-bg');
+const popUp = document.createElement('div');
+
+const createComments = (comments) => {
+  const commentMainDiv = document.querySelector('.comment');
+
+  for (let i = 0; i < comments.length; i += 1) {
+    const commentDiv = document.createElement('div');
+    commentDiv.className = 'comment-item';
+    commentDiv.innerHTML = `
+     <p class="commentDate">${comments[i].creation_date}</p>
+     <p class="commenterName">${comments[i].username}:</p>
+     <p class="commentContent">${comments[i].comment}</p>
+     `;
+    commentMainDiv.appendChild(commentDiv);
+  }
+};
```

Comment:
It would be better if you would more javascript method like (map, filter, forEach,find). In this case you  can use ForEach Method .

Reply:
@soesandarwin2201 Thank you so much for your comments. In this place, I had to use indexes so that I couldn`t use other methods.